project_8.html

```diff
@@ -0,0 +1,793 @@
+								PinHeader_1x02_P2.54mm_Vertical
+							</td>
+						</tr>
+						<tr>
+							<td>x1</td>
+							<td>
+								NE555P IC
+							</td>
+							<td>
+								NE555P
+							</td>
+							<td>
+								Package_DIP
+							</td>
+							<td>
+								DIP-8_W7.62mm
+							</td>
+						</tr>
+
+						<tr>
+							<td>x1</td>
+							<td>
+								Resistor, 1kΩ
+							</td>
+							<td>
+								R_US
+							</td>
+							<td>
+								Resistor_THT
+							</td>
+							<td>
+								R_Axial_DIN0207_L6.3mm_D2.5mm_P7.62mm_Horizontal
+							</td>
+						</tr>
+						<tr>
+							<td>x1</td>
+							<td>
+								Resistor, 470kΩ
+							</td>
+							<td>
+								R_US
+							</td>
+							<td>
+								Resistor_THT
+							</td>
+							<td>
+								R_Axial_DIN0207_L6.3mm_D2.5mm_P7.62mm_Horizontal
+							</td>
+						</tr>
+						<tr>
+							<td>x1</td>
+							<td>
+								LED, 3.2V
+							</td>
+							<td>
+								LED
+							</td>
+							<td>
+								LED_THT
+							</td>
+							<td>
+								LED_D3.00mm
+							</td>
+						</tr>
+						<tr>
+							<td>x1</td>
+							<td>
+								Capacitor, 1µF
+							</td>
+							<td>
+								C
+							</td>
+							<td>
+								Capacitor_THT
+							</td>
+							<td>
+								CP_Radial_D4.0mm_P2.00mm
+							</td>
+						</tr>
+					</tbody>
+				</table>
+				<!-- End Part Table -->
+
+				<h2 class="mb-3 fw-bold" id="pj-heading-schematics">Schematics</h2>
+				<div class="accordion mb-5" id="schemAccordian">
+					<div class="accordion-item">
+						<h2 class="accordion-header">
+							<button class="accordion-button" type="button" data-bs-toggle="collapse"
+								data-bs-target="#schemCollapse1" aria-expanded="true" aria-controls="schemCollapse1">
+								Schematic A
+							</button>
+						</h2>
+						<!--Pin to page?-->
+						<div id="schemCollapse1" class="accordion-collapse collapse show"
+							data-bs-parent="#schemAccordian">
+							<div class="accordion-body text-center">
+								<img src='./assets/images/schematics/ops_schem_proj8.png' class="pj-img-fluid mx-auto"
+									width="400px">
+							</div>
+						</div>
+					</div>
+				</div>
+
+				<h2 class="mb-3 fw-bold" id="pj-heading-instructions">Instructions</h2>
+				<p class="mb-3">
+					Each checkpoint guides you through the process of designing the 555 Blinker circuit, which should
+					look something like
+					this upon completion:
+				</p>
+				<div>
+					<img class="img-fluid my-4" src=".\assets\images\project_8\ops_project_8_kicad_completed.png">
+				</div>
+				<p class="mb-3">
+					<strong>Make sure to save your progress frequently.</strong> Nothing is more infuriating than trying
+					to recover lost data and lost time.
+				</p>
+				<p class="mb-3">
+					<strong>You should also do yourself the favor of using a mouse.</strong> Suffice to say, KiCad is
+					unkind to touchpad users (and if you are using a pointing stick, something is wrong with you.)
+				</p>
+				<p class="mb-3">
```

Comment:
Change to mb-5. Same applies to all other headers. The preceding block element should have a bottom margin of 5rem.

Reply:
this is a little unclear to me. Do you mean that all <p> should have an mb-5?